docs/core-concepts/dotnet-core.md

```diff
@@ -0,0 +1,145 @@
+
+People commonly ask how .NET Core is implemented, in order to support multiple operating systems. They typically ask if there are separate implementations or if [conditional compilation](https://en.wikipedia.org/wiki/Conditional_compilation) is used. It's both, with a strong bias towards conditional compilation.
+
+You can see in the chart below that the vast majority of [CoreFX](https://github.com/dotnet/corefx) is platform-neutral code that is shared across all platforms. Windows and unix implementations are similar in size. Windows has a larger implementation since CoreFX implements some Windows-only features, such as [Microsoft.Win32.Registry](https://github.com/dotnet/corefx/tree/master/src/Microsoft.Win32.Registry) and [System.Security.Principal.Windows](https://github.com/dotnet/corefx/tree/master/src/System.Security.Principal.Windows) but does not yet implement any unix-only concepts. You will also see that the majority of the Linux and OS X implementations are shared across a unix implementation, while the Linux- and OS X-specific implementations are roughly similar in size.
+
+![CoreFX: Lines of Code per Platform](../corefx-platforms-loc.png)
+
+There are a mix of platform-specific and platform-neutral libraries in .NET Core. You can see the pattern in a few examples:
+
+- [CoreCLR](https://github.com/dotnet/coreclr) is platform-specific. It's built in C/C++, so is platform-specific by construction.
+- [System.IO](https://github.com/dotnet/corefx/tree/master/src/System.IO)) and [System.Security.Cryptography.Algorithms](https://github.com/dotnet/corefx/tree/master/src/System.Security.Cryptography.Algorithms) are platform-specific, given that the storage and cryptography APIs differ significantly on each OS. 
+- [System.Collections](https://github.com/dotnet/corefx/tree/master/src/System.Collections)) and [System.Linq](https://github.com/dotnet/corefx/tree/master/src/System.Linq)) are platform-neutral, given that they create and operate over data structures.
+
+Comparisons to other .NET Platforms
+===================================
+
+It is perhaps easiest to understand the size and shape of .NET Core by comparing it to existing .NET platforms. 
+
+Comparison with .NET Framework
+------------------------------
+
+The .NET platform was first announced by Microsoft in 2000 and then evolved from there. The .NET Framework has been the primary .NET product produced by Microsoft during that 15+ year span. 
+
+The major differences between .NET Core and the .NET Framework: 
+
+- **App-models** -- .NET Core does not support the .NET Framework app-models, in part because many of them are built on Windows technologies, such as WPF (built on top of DirectX). The console and ASP.NET Core app-models are supported by both .NET Core and .NET Framework. 
```

Comment:
"does not support the .NET Framework app-models" -> "does not support all of the .NET Framework app-models".